src/Symfony/Component/Serializer/Mapping/Loader/AnnotationLoader.php

```diff
@@ -100,6 +100,11 @@ public function loadClassMetadata(ClassMetadataInterface $classMetadata)
                 continue;
             }
 
+            $getAccessor = preg_match('/^(get|)(.+)$/i', $method->name);
```

Comment:
This regex looks "fishy": ignoring capturing groups (as matches are not used), it's equivalent to `/^(get)?.+$/i` i.e. just `/^.+$/` i.e. will match *any* method (not only getters)

Reply:
I've filed #47255 to fix this.

Reply:
I missed that. A copy paste error. Thanks @jsor for fixing